docs/get.php

```diff
@@ -1,5 +1,5 @@
 <?php
-require_once(__DIR__ . '/api_read/api.php');
+require_once(__DIR__ . '/../api/read/api.php');
```

Comment:
This whole file should really be moved somewhere else (api/read, probably) and the readme updated